### PR TITLE
Provide key-value store id schema(#558)

### PR DESCRIPTION
**Pull request by AndresMurguido001**
Id like to help with this issue. I followed the advice posted in the original issue just to start. Im wondering what exactly is needed to resolve this. I only added the tests to figure out the direction i should be heading. Any advice would be helpful. Thanks !

### CODE REVIEW

**Comment by glasserc** [comment]
Refs #558. Great! Sorry I didn't respond to your question on the issue but it seems like you're on the right track. The goal is to export another idSchema similar to `createUUIDSchema`. So I think everything is correct about your patch except where you put the schema -- it shouldn't be in the tests, but maybe in the same place as `createUUIDSchema`. Also it would be good for there to be a docstring saying something about how this might be useful if you don't want IDs to be generated and giving a use case.

**Comment by AndresMurguido001** [comment]
Thanks for the feedback! I removed it from the tests file and added everything you mentioned. I hope this helps!

File: src/collection.js [hunk 1]
```diff
@@ -134,6 +134,29 @@ function createUUIDSchema() {
...
+ * Using this IDSchema requires you to set a property as the id.
+ * This will be the property used to retrieve this record.
+ *
+ * @example
+ * const exampleCollection = db.collection("example", { idSchema: createKeyValueIdSchema() })
+ * await exampleCollection.create({ title: "How to tie a tie", favoriteColor: "blue", id: "user123" }, { useRecordId: true })
+ * await exampleCollection.getAny("user123")
+ *
+ */
+
```
Comment (glasserc) [inline src/collection.js]: I think we can get rid of this extra blank line.

```suggestion
```

File: src/collection.js [hunk 2]
```diff
@@ -134,6 +134,29 @@ function createUUIDSchema() {
...
+export function createKeyValueIdSchema() {
```
Comment (glasserc) [inline src/collection.js]: Nit: can we call it maybe `createKeyValueStoreIdSchema`? This is a term of art with a specific meaning, see https://en.wikipedia.org/wiki/Key-value_database.

**Pull request merged**